File: app/src/main/java/com/marcinorlowski/fonty/demo/Application.java

package com.marcinorlowski.fonty.demo;

import com.marcinorlowski.fonty.Fonty;

/**
 ******************************************************************************
 *
 * Copyright 2013-2020 Marcin Orlowski <github@MarcinOrlowski.com>
 *
 * Licensed under the Apache License 2.0
 *
 ******************************************************************************
 */
public class Application extends android.app.Application {

	@Override
	public void onCreate() {
		super.onCreate();

		Fonty.context(this)
//            .fontDir("otherFolder")
//            .typefaceFallback(false)
			 .normalTypeface("Exo-Regular.ttf")
			 .italicTypeface("Aramis-Italic.ttf")
			 .boldTypeface("Capture_it.ttf")
			 .build();
	}

}

File: app/src/main/java/com/marcinorlowski/fonty/demo/Application.kt
package com.marcinorlowski.fonty.demo

/** ******************************************************************************
 *
 * Fonty - Android custom fonts made easy
 *
 * @author      Marcin Orlowski <mail (#) marcinOrlowski (.) com>
 * @copyright   2013-2022 2022 Marcin Orlowski
 * @license     https://opensource.org/licenses/Apache-2.0
 * @link        https://github.com/MarcinOrlowski/fonty
 *
 ***************************************************************************** **/

import android.app.Application
import com.marcinorlowski.fonty.Fonty.Companion.context

class Application : Application() {
    override fun onCreate() {
        super.onCreate()
        context(this)
//              .fontDir("otherFolder")
//              .typefaceFallback(false)
                .italicTypeface("Aramis-Italic.ttf")
                .normalTypeface("Exo-Regular.ttf")
                .boldTypeface("Capture_it.ttf")
                .build()
    }
}
